app/vmctl/flags.go

```diff
@@ -320,10 +320,11 @@ var (
 )
 
 const (
-	vmNativeFilterMatch     = "vm-native-filter-match"
-	vmNativeFilterTimeStart = "vm-native-filter-time-start"
-	vmNativeFilterTimeEnd   = "vm-native-filter-time-end"
-	vmNativeStepInterval    = "vm-native-step-interval"
+	vmNativeFilterMatch             = "vm-native-filter-match"
+	vmNativeFilterTimeStart         = "vm-native-filter-time-start"
+	vmNativeFilterTimeEnd           = "vm-native-filter-time-end"
+	vmNativeFilterTimeReverseChunks = "vm-native-filter-time-reverse-chunks"
```

Comment:
why you call it "chunks"? could it be just `vm-native-filter-time-reverse`?

Reply:
renamed this flag